src/Components/FlexBox/index.tsx

```diff
@@ -0,0 +1,20 @@
+// Libraries
+import React, {Component} from 'react'
+
+// Components
+import {FlexBoxRoot, FlexBoxProps} from './FlexBox'
+import {FlexBoxChild} from './FlexBoxChild'
+
+export class FlexBox extends Component<FlexBoxProps> {
+  public static readonly displayName = 'FlexBox'
+
+  public static FlexBox = FlexBoxRoot
+  public static Child = FlexBoxChild
+
+  render() {
+    return <FlexBoxRoot {...this.props} />
+  }
+}
+
+export {FlexBoxProps, FlexBoxRef} from './FlexBox'
```

Comment:
This is also being exported from FlexBoxRoot so it's exported twice from the barrel. Check all instances of this.